mage_ai/data_preparation/models/pipeline.py

```diff
@@ -128,6 +127,10 @@ def version(self):
     def version_name(self):
         return f'v{self.version}'
 
+    @property
+    def pipeline_spark_config(self):
```

Comment:
do you need this property method? is it the same with self.spark_config?

Reply:
Good catch: removed it.